### ISSUE TITLE
Precompiled pthread .lib and .dll files in GIT

### ISSUE THREAD

**Issue by msoos** [stp/abc#2]
```
pthreadVC2.dll  pthreadVC2.lib
```

under both `lib/x64` and `lib/x86`. There are two problems with these: they are precompiled and they are not libabc but pthread. They should go I believe.

**Comment by TrevorHansen** [comment]
Yeah, it's weird to include these files. Feel free to remove them if you want. I'm going to leave them for now. Don't seem to be causing any problems, and we're racing ahead with removing our dependency on ABC anyway.
